casepro/contacts/models.py

```diff
@@ -245,12 +256,12 @@ def as_json(self, full=True):
         """
         Prepares a contact for JSON serialization
         """
-        result = {'uuid': self.uuid, 'name': self.name}
+        result = {'id': self.pk, 'name': self.get_display_name()}
```

Comment:
They've only ever been able to see the first 6 chars of the UUID. We added links to the contact in rapidpro on the inbox for admins - with this change those links take a user to the casepro contact page, and from there they can jump to rapidpro by clicking "View External".